WSFrontend/org.openl.rules.ruleservice.ws.storelogdata.hive/src/org/openl/rules/ruleservice/storelogdata/hive/HiveOperations.java

```diff
@@ -171,13 +137,17 @@ public void setCreateTableEnabled(boolean createTableEnabled) {
         this.createTableEnabled = createTableEnabled;
     }
 
-    public void setConnectionURL(String connectionURL) {
-        this.connectionURL = connectionURL;
+    public HikariDataSource getHiveDataSource() {
+        return hiveDataSource;
+    }
+
+    public void setHiveDataSource(HikariDataSource hiveDataSource) {
+        this.hiveDataSource = hiveDataSource;
     }
 
     private static String extractSqlQueryForEntity(Class<?> entityClass) throws IOException {
         InputStream inputStream = entityClass
-            .getResourceAsStream("/" + entityClass.getName().replaceAll("\\.", "/") + ".sql");
+                .getResourceAsStream("/" + entityClass.getName().replaceAll("\\.", "/") + ".sql");

```

Comment:
Could this SQL query file contain the comments? Because in Cassandra services, there is an operation that removes the comments in the SQL query file.

Reply:
This feature was added to master branch